src/js/components/Sort.js

```diff
@@ -30,15 +30,17 @@ export default class Sort extends Component {
 
   render () {
     const { options, value, direction, ...props } = this.props;
-    let label;
+
+    let selectedOption;
     if (value) {
-      label = options.filter(option => option.value === value)[0].label;
+      selectedOption = options.filter(option => option.value === value);
```

Comment:
I wonder if this is a breaking change. I'm not very familiar with Sort or grommet-addons, but by inspecting this code I believe this is a breaking change.

Having break changes is definitely ok, but we need to be careful when we release to make sure we describe what needs to change from the caller's perspective.

I think `grommet-docs` uses Sort as an example. Have you tested it there? Did you have to do any changes?